WooCommerce/Classes/POS/Presentation/Item Search/POSSearchTextFieldStyle.swift

```diff
@@ -0,0 +1,58 @@
+import SwiftUI
+
+/// Text field style for search fields that includes a magnifier icon and clear button
+struct POSSearchTextFieldStyle: TextFieldStyle {
+    private let focused: Bool
+    @Binding private var searchTerm: String
+    @ScaledMetric private var searchFieldHeight: CGFloat = 56.0
+
+    init(focused: Bool, searchTerm: Binding<String>) {
+        self.focused = focused
+        self._searchTerm = searchTerm
+    }
+
+    func _body(configuration: TextField<Self._Label>) -> some View {
+        configuration
+        .textFieldStyle(WooRoundedBorderTextFieldStyle(
+            focused: focused,
+            focusedBorderColor: .posPrimary,
+            unfocusedBorderColor: .posSurfaceBright,
+            backgroundColor: .posSurfaceBright,
+            cornerRadius: POSCornerRadiusStyle.medium.value,
+            height: searchFieldHeight,
+            content: { configuration in
+                AnyView(
+                    HStack(spacing: POSSpacing.small) {
```

Comment:
Comparing with designs, it looks like a few things should be checked and possibly adjusted:

- Space between the back button and the search field
- Search icon size
- Leading and trailing spaces around the icon

<img width="904" alt="icon size" src="https://github.com/user-attachments/assets/df1485df-18df-452f-8539-1311d8d0a682" />

Reply:
Yep, good point. I thought I'd picked most of those definitions out of Figma, but I'll go back and double check them all. Thanks for the detailed check.

Reply:
All sorted. The back button needed a bit of a padding hack, but I wanted to keep the definition for the button inside the search view, so I think it was worth it.